### PR TITLE
Add poky support

### PR DESCRIPTION
**Pull request by mortenvp**

### CODE REVIEW

File: cxx_mkspecs/gxx_mkspecs.py [hunk 1]
```diff
@@ -444,3 +446,35 @@ def cxx_openwrt_gxx53_mips(conf):
     # Note: libstdc++ might not be available on the target platform
     # Statically link with the C++ standard library
     conf.env['LINKFLAGS'] += ['-static-libstdc++']
+
+
+@conf
+def cxx_poky_gxx63_arm7(conf):
```
Comment (petya2164) [inline cxx_mkspecs/gxx_mkspecs.py]:
```suggestion
def cxx_poky_gxx63_armv7(conf):
```
We use the armv7 postfix in the other mkspecs, so let's make this consistent.

File: wurf_cxx_mkspec.py [hunk 2]
```diff
@@ -108,6 +108,11 @@ def options(opt):
         '--emscripten_path', default=None, dest='emscripten_path',
         help="Path to the Emscripten compiler (em++)")
 
+    opts.add_option(
+        '--poky_sdk_path', default=None, dest='poky_sdk_path',
+        help="Path that should be searched when looking for the toolchaing e.g."
```
Comment (petya2164) [inline wurf_cxx_mkspec.py]: Let's make this help text more informative: explain what toolchain this is!

**Comment by mortenvp** [comment]
@petya2164 good changes - what do you think?

File: wurf_cxx_mkspec.py [hunk 3]
```diff
@@ -108,6 +108,11 @@ def options(opt):
         '--emscripten_path', default=None, dest='emscripten_path',
         help="Path to the Emscripten compiler (em++)")
 
+    opts.add_option(
+        '--poky_sdk_path', default=None, dest='poky_sdk_path',
+        help="Configure toolchain (cross compiler etc.) for the Yocto based "
```
Comment (petya2164) [inline wurf_cxx_mkspec.py]:
```suggestion
        help="Path to the Yocto-based cross-compiler toolchain for the  "
```
I tried to harmonize the description with the other sdk path options above.

**Comment by mortenvp** [comment]
@petya2164 should we merge?

**Comment by petya2164** [comment]
Sure, I can merge and release this now!

**Pull request merged**